### DISCUSSION TITLE
How can i specify Ephemeral Storage Size?

### DISCUSSION THREAD

**Discussion by kostiantyn-pravdiuk** [serverless/serverless#11476] [Q&A]
### Is there an existing issue for this?

- [X] I have searched existing issues, it hasn't been reported yet

### Use case description

I'm trying to set a custom value for phemeralStorageSize. I tried to do as it is written in the documentation but I get a warning in the console. How can this value be specified?
<img width="486" alt="Знімок екрана 2022-10-26 о 15 15 24" src="https://user-images.githubusercontent.com/106402189/198023608-cebd63d6-04a1-48e9-9961-f5681d4f0b53.png">
<img width="1002" alt="Знімок екрана 2022-10-26 о 15 15 32" src="https://user-images.githubusercontent.com/106402189/198023610-7883ad3a-8742-4766-952c-0be1f7fde84f.png">

**Comment by medikoo** [comment]
@kostiantyn-pravdiuk ensure to rely on latest version of the Framework
